CONTRIBUTING.md

```diff
@@ -31,19 +103,77 @@ Code contributions should be submitted in the form of a pull request. Here are t
 ```
 git clone <the url of your fork>
```

Comment:
i think we could ditch this and just leave it at "fork the code" with the external link to forking.